gst/audio/gst_audio_meta.go

```diff
@@ -0,0 +1,36 @@
+package audio
+
+/*
+#include "gst.go.h"
+*/
+import "C"
+
+import (
+	"unsafe"
+
+	"github.com/go-gst/go-gst/gst"
+)
+
+type AudioMeta struct {
+	ptr *C.GstAudioMeta
+}
+
+func wrapMetaFull(ptr *C.GstAudioMeta) *AudioMeta {
+	meta := &AudioMeta{ptr}
+	return meta
+}
+
+func BufferAddAudioMeta(buffer *gst.Buffer, info *Info, samples int /*, offsets *[]int*/) *AudioMeta {
+	/*gSizeOffsets := C.gsize(unsafe.Sizeof(unsafe.Pointer(offsets)))*/
+	// if you pass in NULL as the last param then gstreamer assumes things are tightly packed...
+	// and they are for me so this makes things work... but obviously would be nice for it to work
+	// properly
```

Comment:
```suggestion
// offsets is not yet implemented, always pass `nil` or this will panic
func BufferAddAudioMeta(buffer *gst.Buffer, info *Info, samples int, offsets []int) *AudioMeta {
       if offsets != nil { 
           panic("offsets is not implemented")
       }

	/*gSizeOffsets := C.gsize(unsafe.Sizeof(unsafe.Pointer(offsets)))*/
	// if you pass in NULL as the last param then gstreamer assumes things are tightly packed...
	// and they are for me so this makes things work... but obviously would be nice for it to work
	// properly
```

Doing so prevents a breaking change when we implement it.

(please use an IDE to apply, I typed this in the browser, so formatting may be off)